homework/homework2.playground/Contents.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+let a = 2.5
+let b = Int(a)
+print(b)
+let c = a - Double(b)
+print(c)
```

Comment:
As an improvement, I suggest we start using the correct properties naming from the start. So to follow that we can rename `a` property to `firstNumber`, `b` property to `firstNumberWhole`, `c` to `firstNumberFractional`, `a1` to `secondNumber` and so on